client/state/data-layer/wpcom/sites/rewind/schema.js

```diff
@@ -151,6 +151,8 @@ export const active = stateSchema( 'active', {
 				threats: { type: threat },
 			},
 		},
+		last_backup_when: { type: 'string' },
```

Comment:
Jetpack is expecting `last_good_backup`.

Reply:
🤦‍♂ fixed